activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -883,6 +883,12 @@ def left_outer_joins!(*args) # :nodoc:
     #    PriceEstimate.where(estimate_of: treasure)
     #    PriceEstimate.where(estimate_of_type: 'Treasure', estimate_of_id: treasure)
     #
+    # Hash conditions may also be specified in a tuple-like syntax. Hash keys may be a list of columns,
+    # with a list of tuples as values.
```

Comment:
`array` seems to be more commonly used in these docs.
```suggestion
    # Hash conditions may also be specified in a tuple-like syntax. Hash keys may be an array of columns,
    # with an array of tuples as values.
```